neo/SmartContract/Native/Tokens/NeoToken.cs

```diff
@@ -41,6 +41,8 @@ protected override StackItem Main(ApplicationEngine engine, string operation, VM
                     return RegisterValidator(engine, args[0].GetByteArray());
                 case "vote":
                     return Vote(engine, new UInt160(args[0].GetByteArray()), ((VMArray)args[1]).Select(p => p.GetByteArray().AsSerializable<ECPoint>()).ToArray());
+                case "getRegisteredValidators":
+                    return GetRegisteredValidators(engine).Select(p => new Struct(new StackItem[] { p.PublicKey.ToArray(), p.Votes })).ToArray();
```

Comment:
`Struct` or `Array`? due that is only for return maybe we should use an `Array` because have less encapsulation

Reply:
It is an array of struct.

Reply:
But do think that is not better ?

```suggestion
                    return GetRegisteredValidators(engine).Select(p => new Array(new StackItem[] { p.PublicKey.ToArray(), p.Votes })).ToArray();
```

Reply:
This is a contract, and may be called from another contract. In that contract, it can define a struct:

``` c#
struct RegisteredValidator
{
    byte[] pubkey;
    BigInteger votes;
}
```

How do you deal with this via `Array`?